ynput/ayon-unreal
# Inventory Action for connecting animation sequence to level sequence loaded from the layout product type

## Changelog Description
This PR is to add inventory action which allows the animation sequence from the animation product type connecting to the level sequence which includes skeletal mesh with the rig. 

## Additional info
This is only applicable when you loaded the level sequence with layout product type. 
There is some adjustment on adding frame range data into the container. Users need to remove and re-load the animation sequence before testing this ticket.

https://github.com/user-attachments/assets/632b35ee-18b7-4f54-a87d-d02ab21e384e

There is some limitation for the current UE workflow which needs users to use action to connect the animation when they are updating the animations.


## Testing notes:
1. Load Layout which stores the transformation data of the rig
2. Load animation which stores the animation data related to the rig
3. Ayon -> Manage
4. Select the loaded layout instance and the loaded animation instance
5. Right click -> Action -> Connect Alembic Animation to Level Sequence/Connect Fbx Animation to Level Sequence
6. Click the level sequence located in the layout folder
7. The related animation track should be added and bind to the animation sequence 
![image](https://github.com/user-attachments/assets/fcae7d09-2952-4fbc-b862-e2648b716ad0)

Updating/setting version of the animation
1. Updating your animation
2. Select the loaded layout instance and the loaded animation instance
3. Animation sequence would update the animation.
